src/api/manufacturers.tsx

```diff
@@ -106,15 +106,15 @@ const editManufacturer = async (
     .then((response) => response.data);
 };
 
-export const useEditManufacturer = (): UseMutationResult<
+export const usePatchManufacturer = (): UseMutationResult<
   Manufacturer,
   AxiosError,
   ManufacturerPatch
 > => {
   const queryClient = useQueryClient();
   return useMutation({
     mutationFn: (manufacturer: ManufacturerPatch) =>
```

Comment:
Now that the types match very closely, I would kind of prefer this to take an id and the patch as id isn't really part of the patch schema. But it could also be more of an audit thing and would be best addressed after the form changes.

Reply:
I get what you mean but the way it is currently done now doesn't decrease readability. The main type that we need to make sure is exactly as the backend is the output type (Manufacturer,System) all of the other (post and patch) ones should contain the information we need the react query hook to send the information

Reply:
Yeah that's fine, its just me being picky with the definitions. Saying its the api schema implies to me it should be what the api takes, not the front end function that sends it otherwise it would be in the app.types as its an app type not an api type. But I am fine with it here now.